common/technology/technologies/30_society.txt

```diff
@@ -71,6 +71,9 @@ currency_standards = {
 	era = era_1
 	texture = "gfx/interface/icons/invention_icons/currency_standards.dds"
 	category = society
+	modifier = {
+		building_cash_reserves_mult = 0.1
```

Comment:
You know this is in effect a slight debuff for investment pool as well as and enlargement the government max credit right?